pkg/store/consul/consulutil/safe.go

```diff
@@ -7,13 +7,8 @@ import (
 	"runtime"
 
 	"github.com/hashicorp/consul/api"
-
-	"github.com/square/p2/pkg/util/param"
 )
 
-// Show detailed error messages from Consul (use only when debugging)
-var showConsulErrors = param.Bool("show_consul_errors_unsafe", false)
-
 // KVError encapsulates an error in a Store operation. Errors returned from the
 // Consul API cannot be exposed because they may contain the URL of the request,
```

Comment:
I'm kind of wishing this comment would change to reflect the new reality.

... perhaps the error also no longer needs to be called `UnsafeError`...?